webapp/public/javascripts/angular/data-series/registration.js

```diff
@@ -1574,7 +1589,7 @@ angular.module('terrama2.dataseries.registration', [
         }
 
         if ($scope.filter.filterArea == $scope.filterTypes.AREA.value) {
-          var boundedForm = angular.element('form[name="boundedForm"]').scope().boundedForm;
+          var boundedForm = angular.element('form[name="filterForm.boundedForm"]').scope().filterForm.boundedForm;
```

Comment:
It should be replaced by TerraMA² angular service ```FilterForm``` instead use angular jqlite selector.
```javascript
if (FilterForm.boundedForm.$invalid) {
  // do some stuff
}
```